lib/src/lib_edit_address/ui_components/address_for_correspondence_bottom_sheet.dart

```diff
@@ -29,37 +31,40 @@ void showAppAddressForCorrespondence<T extends PickerItemModel>(
   final EditFieldType editAddressFieldType = EditFieldType.editfield,
   final Widget Function(ErrorModel?)? editContactAddressErrorBuilder,
   final SearchCountryCustomBuilders<T>? searchCountryCustomBuilders,
-}) =>
-    showModal(
-      configuration: ModalConfiguration(
-        fullScreen: configuration.fullScreen,
-        heightFactor: configuration.heightFactor,
-        isDismissible: configuration.isDismissible,
-        safeAreaBottom: false,
-      ),
-      context: context,
-      builder: (ctx) => EditAddressPage.withDependencies<T>(
-        context,
-        buttonText: buttonText,
-        headerText: headerText,
-        addressModel: addressModel,
-        cityErrorMapper: cityErrorMapper,
-        addressErrorMapper: addressErrorMapper,
-        validator: validator,
-        countryCustomIcon: countryCustomIcon,
-        editCountryFieldType: editCountryFieldType,
-        cityCustomIcon: cityCustomIcon,
-        editCityFieldType: editCityFieldType,
-        addressCustomIcon: addressCustomIcon,
-        editAddressFieldType: editAddressFieldType,
-        searchCountryService: searchCountryService,
-        editAddressLocalizedStrings: editAddressLocalizedStrings,
-        editAddressService: editAddressService,
-        editContactAddressErrorBuilder: editContactAddressErrorBuilder,
-        searchCountryCustomBuilders: searchCountryCustomBuilders,
-      ),
-      onCancelPressed: () => Navigator.of(context).pop(),
-    );
+}) {
+  var editAddressBlocType = context.read<EditAddressBlocType>();
```

Comment:
A couple of lines below you use the `EditAddressPage.withDependencies`, which means that there is no need of injecting the `editAddressBlocType` as a separate prop